bok_choy/a11y/a11y_audit.py

```diff
@@ -167,7 +167,8 @@ def check_for_accessibility_errors(self):
         if audit_results:
             self.report_errors(audit_results, self.url)
 
-    @abstractproperty
+    @property
```

Comment:
` @abc.abstractproperty`

> Deprecated since version 3.3: It is now possible to use property, property.getter(), property.setter() and property.deleter() with abstractmethod(), making this decorator redundant.

https://docs.python.org/3/library/abc.html#abc.abstractproperty